distributions/aws/test/e2e/README.md

```diff
@@ -0,0 +1,51 @@
+
+## Usage
+
+Install dependencies
+```
+python3
+kustomize   # MUST BE VERSION 3.2.0
+eksctl
+kubectl
+
+awscli # configure your default credentials
+
+# install python dependencies
+pip install -r requirements.txt
```

Comment:
Can we include black here and also find a way to make the code formatting for these files automated ?

Reply:
done